models/prices/polygon/prices_polygon_tokens.sql

```diff
@@ -87,5 +87,6 @@ FROM
     ('gtc-gitcoin', 'polygon', 'GTC', 0x3d93f3bc2cb79c31b4df652cd332d84d16317889,18),
     ('gmt-stepn', 'polygon', 'GMT', 0x714db550b574b3e927af3d93e26127d15721d4c2,8),
     ('tel-telcoin', 'polygon', 'TEL', 0xdf7837de1f2fa4631d716cf2502f8b230f1dcc32,2),
-    ('mana-decentraland', 'polygon', 'MANA', 0xa1c57f48f0deb89f569dfbe6e2b7f46d33606fd4,18)
+    ('mana-decentraland', 'polygon', 'MANA', 0xa1c57f48f0deb89f569dfbe6e2b7f46d33606fd4,18),
+    ('work-the-employment-commons-work-token', 'polygon', 'WORK', 0x6002410dda2fb88b4d0dc3c1d562f7761191ea80, 18)
```

Comment:
Looks like there's an error with `WORK`.

Reply:
Yep, seems like even though it was still listed on the website, it was no longer on the api. I have removed that part of the PR.